job_templates/packaging_job.xml.em

```diff
@@ -32,11 +32,13 @@
     ignore_rmw_default=ignore_rmw_default,
     use_connext_debs_default=use_connext_debs_default,
 ))@
-        <hudson.model.BooleanParameterDefinition>
-          <name>CI_TEST_BRIDGE</name>
-          <description>By setting this to True, the tests for the ros1_bridge will be run.</description>
-          <defaultValue>@(test_bridge_default)</defaultValue>
-        </hudson.model.BooleanParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>CI_MIXED_ROS_OVERLAY_PKGS</name>
+          <description>
+A list of packages - separated by spaces - which explicitly have to be built after sourcing the ROS 1 environment.
+All packages listed here have to be available from either the primary or supplemental repos file.
+          </description>
+        </hudson.model.StringParameterDefinition>
```

Comment:
This new parameter will probably need a default value so that the packaging jobs continue to test the bridge.